nextflow.config

```
@@ -51,32 +13,7 @@ profiles{
       region = 'us-east-1'
       batch.cliPath = '/home/ec2-user/miniconda/bin/aws'
     }
-    process{
-      executor = 'awsbatch'
-      queue = 'nextflow-batch-default-queue'
-      // following options from nf-core
-      errorStrategy = { task.attempt < 2 ? 'retry' : 'finish' }
-      maxRetries = 1
-      maxErrors = '-1'
-      memory = { 4.GB * task.attempt}
-
-      withLabel: mem_8 {
-        memory = { 8.GB * task.attempt}
-      }
-      withLabel: cpus_12 {
-        cpus = 12
-        memory = { 24.GB * task.attempt}
-      }
-      withLabel: cpus_8 {
-        cpus = 8
-        memory = { 16.GB * task.attempt}
-      }
-      withLabel: disk_big {
-        queue = 'nextflow-batch-bigdisk-queue'
-      }
-      withLabel: disk_dynamic {
-        queue = { task.attempt < 2 ? 'nextflow-batch-default-queue' : 'nextflow-batch-bigdisk-queue'}
-      }
-    }
+    // batch parameters
+    includeConfig 'config/process.config'
```

Suggesting renaming this file, as the config in it is specific to aws. As we are going to separate this out, then I think it should include all of the aws settings, from `bucketDir = 's3://nextflow-ccdl-data/work'` down, not just the process settings, as many of the settings don't make sense outside of aws.

We could probably figure out a way to pull out the common parts that might still apply outside of AWS, but I don't think that is worth the effort at this time.

We could also move _all_ of the `profiles` block to a separate file, but I kind of like having the available options visible in the main `nextflow.config` file. 

```suggestion
    includeConfig 'config/profile_awsbatch.config'
```